taliforth.asm

```diff
@@ -270,58 +270,168 @@ _nibble_to_ascii:
                 rts
 
 
-find_header_name:
+; =====================================================================
+; HEADER HELPER FUNCTIONS
+;
+; These functions help navigate the variable size headers.
+; Currently nt_to_nt is a subroutine that's shared by several callers.
+; In the current test suite this is called about 2M times
```

Comment:
Discussion about compilation performance mostly driven by cost of walking the NT list.